boards/snps/nsim/arc_v/doc/index.rst

```diff
@@ -0,0 +1,201 @@
+.. _nsim_arc_v:
+
+DesignWare RISC-V nSIM and HAPS FPGA boards
+###########################################
+
+Overview
+********
+
+This platform can be used to run Zephyr RTOS on the widest possible range of Synopsys RISC-V processors in
+simulation with `Designware ARC nSIM`_ or run same images on FPGA prototyping platform `HAPS`_. The
+platform includes the following features:
+
+* RISC-V processor core, which implements riscv32 ISA
+* Virtual serial console (a standard ``ns16550`` UART model)
+
+There are multiple supported board targets for that platform. Some but not all of currently
+available targets are listed below:
+
+* ``nsim_arc_v/rmx100`` - Synopsys RISC-V RMX100 core
+
+.. _board_nsim_arc_v_prop_files:
+
+It is recommended to look at precise description of a particular board target in ``.props``
+files in :zephyr_file:`boards/snps/nsim_arc_v/support/` directory to understand
+which options are configured and so will be used on invocation of the simulator.
+
+.. warning::
+   All nSIM targets are used for demo and testing purposes. They are not meant to
+   represent any real system and so might be renamed, removed or modified at any point.
+
+Programming and Debugging
+*************************
+
+Required Hardware and Software
+==============================
+
+To run single-core Zephyr RTOS applications in simulation on this board,
+either `DesignWare ARC nSIM`_ or `DesignWare ARC Free nSIM`_ is required.
+
+Building & Running Sample Applications
+======================================
+
+Most board targets support building with both GNU and ARC MWDT toolchains, however
+there might be exceptions from that, especially for newly added targets. You can check supported
+toolchains for the board targets in the corresponding ``.yaml`` file.
+
+I.e. for the ``nsim_arc_v/rmx100`` board we can check :zephyr_file:`boards/snps/nsim_arc_v/nsim_arc_v_rmx100.yaml`
+
+The supported toolchains are listed in ``toolchain:`` array in ``.yaml`` file, where we can find:
+
+* **zephyr** - implies RISC-V GNU toolchain from Zephyr SDK. You can find more information about
+  Zephyr SDK :ref:`here <toolchain_zephyr_sdk>`.
+* **cross-compile** - implies RISC-V GNU cross toolchain, which is not a part of Zephyr SDK. Note that
+  some (especially new) board targets may declare ``cross-compile`` toolchain support without
+  ``zephyr`` toolchain support because corresponding target CPU support hasn't been added to Zephyr
+  SDK yet. You can find more information about its usage here: :ref:`here <other_x_compilers>`.
+* **arcmwdt** - implies proprietary ARC MWDT toolchain. You can find more information about its
```

Comment:
arcmwdt isn't yet supported with Zephyr, right? The tools do support ARC-V, but Zephyr makefiles etc. are not yet in place. Remove here or add a 'work in progress' remark?

Reply:
Add "work in progress" label